src/main/java/dev/sigstore/fulcio/client/SigningCertificate.java

```diff
@@ -79,6 +81,10 @@ static SignedCertificateTimestamp decodeSCT(String sctHeader) throws Serializati
         .toSct();
   }
 
+  public boolean hasEmbeddedSct() {
```

Comment:
some javadoc about embedded vs detached SCT would help here.

Reply:
added to the doc on the class.